src/libais/ais.h

```diff
@@ -1566,9 +1566,49 @@ class Ais8_200_10 : public Ais8 {
   Ais8_200_10(const char *nmea_payload, const size_t pad);
 };
 
-// 21 and 22 do not exist
+// vessel_traking_and_tracing_standard_ed1-2_ccnr_23-apr_2013_en.pdf
```

Comment:
Please give a full link to the pdf.  Also it looks like you have a typo:  traking -> tracking

Reply:
Link to *"Vessel Tracking and Tracing Standard for Inland Navigation Edition 1.2"*: http://www.ris.eu/docs/File/536/vessel_traking_and_tracing_standard_ed1-2_ccnr_23-apr_2013_en.pdf

Official link to the last standard: http://www.ris.eu/library/expert_groups/vtt/standards